### DISCUSSION TITLE
Metrics like horizontalrunnerautoscaler_workflow_runs_queued are not available yet

### DISCUSSION THREAD

**Discussion by mohammad-dabour** [actions/actions-runner-controller#2075] [Questions]
Hello Dears,
I can see only 4 metrics on my monitoring system out of the the 13 metrics listed in this code snippet. Seems the recent versions still do not have this change ?. Do you know if its is it going to be available anytime soon?

https://github.com/actions-runner-controller/actions-runner-controller/blob/538e2783d7fde279b84f1ff9351bb1486823660b/controllers/metrics/horizontalrunnerautoscaler.go#L25-L34

**Comment by mumoshu** [comment]
@mohammad-dabour Hey! AFIAK those metrics work only when you're using pull-based scaling. In case you're using webhook-based autoscaling and want workflow job related metrics to be monitored, you'd need to until #2057 gets released as a part of upcoming ARC 0.27.0.

Could you share HorizontalRunnerAutoscaper spec and what the names of the 4 metrics you could see? Thanks!

**Comment by mohammad-dabour** [comment]
@mumoshu thank you, I only  see the following metrics
horizontalrunnerautoscaler_spec_min_replicas
horizontalrunnerautoscaler_spec_max_replicas
horizontalrunnerautoscaler_status_desired_replicas
horizontalrunnerautoscaler_replicas_desired
We're using pull based actually
